pkg/3scale/amp/operator/redis_options_provider.go

```diff
@@ -272,3 +271,37 @@ func (r *RedisOptionsProvider) setTopologySpreadConstraints() {
 		r.options.BackendRedisTopologySpreadConstraints = r.apimanager.Spec.Backend.RedisTopologySpreadConstraints
 	}
 }
+
+func (r *RedisOptionsProvider) systemRedisPodTemplateAnnotations() map[string]string {
+	annotations := make(map[string]string)
+	for k, v := range r.apimanager.Spec.System.RedisAnnotations {
+		annotations[k] = v
+	}
+	annotations["generationID"] = r.getRedisConfigCmResourceVersion()
```

Comment:
maybe better name would be `redisConfigMapResourceVersion` ??

Reply:
Done. Thank you

Reply:
no, you did not. I still see `generationID`

Reply:
oh, I thought that it's about function name : ).
Done now. Thank you